recipes/sqlite3/3.17/build.cmake

```diff
@@ -1,5 +1,5 @@
 
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Does this need to be at least 3.6?

Reply:
I forked this repo and pointed rbuild to it to make sure this works and 3.5 seems to be enough